src/metabase/analytics/prometheus.clj

```diff
@@ -237,11 +237,21 @@
    (prometheus/counter :metabase-notification/send-error
                        {:description "Number of errors when sending notifications."
                         :labels [:payload-type]})
+   (prometheus/histogram :metabase-notification/wait-duration-ms
```

Comment:
I'm open for suggestion of the name here, one pattern I'm considering is to prefix everything with "send":
- send-wait-duration-ms
- send-execute-duration-ms
- send-total-duration-ms

Reply:
I like shorter names more if they are not ambiguous so this seems fine (i'd even be ok with `wait-ms` 😁)